internal/config/generate/Config.toml

```diff
@@ -234,6 +234,7 @@ description = """
 When enabled, the node does not start the authority-claimer service and the Redis server."""
 
 [experimental.CARTESI_EXPERIMENTAL_SUNODO_VALIDATOR_REDIS_ENDPOINT]
+default = ""
```

Comment:
Why an empty default?

Reply:
It's a workaround/compromise. 

Now that there's a 1:1 map from env vars to NodeConfig member variables, `getCartesiExperimentalSunodoValidatorRedisEndpoint()` is called even if `CARTESI_EXPERIMENTAL_SUNODO_VALIDATOR_ENABLED=false` and we'll get an error if there's no value.

I couldn't find a solution that didn't involve a lot more code, so I went this not-so-elegant route. This is one of the reasons I think another issue is needed to better adapt the `config` package API.

Reply:
I will take a better look on this now. These "logic" about when to use or how to use the configs are spread out at some functions outside the Config Package.